src/btree.c

```diff
@@ -3157,6 +3156,109 @@ btree_count_in_range_by_iterator(cache        *cc,
    btree_iterator_deinit(&btree_itor);
 }
 
+/* Print offset table entries, 4 entries per line, w/ auto-indentation. */
+static void
+btree_print_offset_table(platform_log_handle *log_handle, btree_hdr *hdr)
+{
+   platform_log(log_handle, "-------------------\n");
+   platform_log(log_handle, "Offset Table num_entries=%d\n", hdr->num_entries);
+
+   uint64 nentries = btree_num_entries(hdr);
+   char   fmtstr[30];
+   snprintf(fmtstr,
+            sizeof(fmtstr),
+            "[%%%s] %%-8u",
+            ((nentries < 10)     ? "d"
+             : (nentries < 100)  ? "2d"
+             : (nentries < 1000) ? "3d"
+                                 : "4d"));
```

Comment:
This feels unnecessary and tedious. Why not just use `3d` (1000s of entries feels unlikely to happen soon)?

Reply:
Based on our off-line slack chat, changed this inline code to a macro, `PRInd()` defined in `util.h`.
Macro evaluates to one of the "d", "2d", etc. print format specifiers.

Reply:
I like the macro, but can we call it something more in line with our naming conventions, like `DECIMAL_STRING_WIDTH`?